src/components/TopBar.js

```diff
@@ -328,16 +329,19 @@ const TopBar = ({ showMenu = true, title, href, children }) => {
       ])
     ])
   }
+  const mainRef = useRef()
 
-  return h(Fragment, [
+  return div({
+    role: 'banner'
+  }, [
```

Comment:
I'd tell you to just collapse this to one line, for simplicity

Reply:
ok